ocs_ci/ocs/utils.py

```diff
@@ -677,6 +680,24 @@ def setup_ceph_toolbox(force_setup=False):
             'json --patch  \'[{ "op": "replace", "path": '
             '"/spec/enableCephTools", "value": true }]\''
         )
+    # use the existing toolbox but fix the keyring to talk to
+    # external cluster
+    independent_mode = ocsci_config.DEPLOYMENT.get("independent_mode")
+    if independent_mode:
+        keyring_dict = ocsci_config.INDEPENDENT_MODE.get("admin_keyring")
+        log.info("Setting up keyring on toolbox for independent mode")
+        ceph_toolbox = get_pod_name_by_pattern('rook-ceph-tools', namespace)
```

Comment:
so if I understand correctly the control flow, when we are deploying OCS for external cluster , at this point in time we will not be having ```rook-ceph-tools``` existing right ? OR you assume that we deploy it first without keyring and then update ?

Reply:
yeah correct, the call here should setup toolbox, I haven't seen the yaml in detail but it should help setup the pod with client binaries and we update the keyring.